### ISSUE TITLE
Add filter for type

### ISSUE THREAD

**Issue by ecsmyth** [mozilla/probe-dictionary#100]
As a user searching for a probe, I want a filter the list of results based on the type of probe, so that I can more quickly find probes of interest among dozens or hundreds that match my current results.

**Comment by spasovski** [comment]
Good call, this will likely be rolled into the probe search integration already in progress.
